src/openforms/prefill/contrib/haalcentraal/plugin.py

```diff
@@ -113,39 +113,20 @@ def get_co_sign_values(cls, identifier: str) -> Tuple[Dict[str, Any], str]:
         )
 
     def check_config(self):
-        check_bsn = "111222333"
         try:
             config = HaalCentraalConfig.get_solo()
             if not config.service:
                 raise InvalidPluginConfiguration(_("Service not selected"))
 
             client = config.service.build_client()
-            client.retrieve(
+            client.list(
                 "ingeschrevenpersonen",
-                burgerservicenummer=check_bsn,
                 request_kwargs=self.request_kwargs,
+                params={"burgerservicenummer": "123"},
             )
```

Comment:
Requirement:

> refactor to do a GET-requests that is invalid to simple check the connection/ssl

**Mistake**

You are not doing an invalid GET request.

**Rationale**

We can not make "real" calls _as described in the ticket_. At best, we can do connection tests.